Req: When user selected a tab, it will display title, description, and code dynamically

Tarachand90 opened the pull request:
- When user clicked a tab, the content for that tab should be displayed dynamically.

- it will display title, description, and code for the selected tab and we fetched this from
EXAMPLES object using the tabContent state.

Tarachand90 committed:
Req: When user selected a tab, it will display
title, description, and code dynamically.

- When user clicked a tab, the content for that
tab should be displayed dynamically.

- it will display title, description, and code
for the selected tab and we fetched this from
EXAMPLES object using the tabContent state.

DebaKaran commented on src/App.tsx:
```diff
@@ -21,7 +22,28 @@ function App() {
           <h2>Examples</h2>
           <TabMenu onTabSelect={onClickHandler} />
-          <span>{tabContent}</span>
+          <div id="tab-content">
```
Move this code into a different component.

DebaKaran reviewed (commented):
Move it to diff component

Tarachand90 committed:
- Incorporated the review comments.

- Created a new component and removed the
corresopnding details from App.tsx file.

DebaKaran reviewed (approved):
Approved!
